Moved from DEB-based nfqueue 0.5 to NetfilterQueue v0.8.1

moonbaseDelta opened the pull request:
Now it should work fine on different Linux distros. Checked on Arch and CentOS7

GingerSauce committed:
Moved from DEB-based nfqueue 0.5 to NetfilterQueue v0.8.1, now should work fine on different Linux distros

GingerSauce committed:
Indentation troubles handled

GingerSauce committed:
Fixed some bugs. Also now it's not blocking ssh and other protocols

GingerSauce committed:
Safe iptables' target removal

GingerSauce committed:
Accidental file removed from repo

GingerSauce committed:
Now capable to run on multiple network interfaces simultaniously

GingerSauce committed:
Fixed nfqueue id's for multiple devices

ezbik commented:
>Without using -i option, I get error "bad value for --queue-num or out of range (0-65535)"

@janisblaus look at the code, it implies your main network card is `eth0` and it tries to calculate NFQUEUE value based on paths in /sys/class/net/YOUR_NETWORK_CARD/

ezbik commented:
@moonbaseDelta Your fork is awesome! it seems @segofensiva abandoned his development, so can you open Issues on your repo? I have some ideas:
- allow user to specify NFQUEUE number instead of calculating it from NIC number in /sys/\*
- print out NFQUEUE number 
- allow user to skip iptables actions ( would be useful on multi-NIC systems or there where Iptables rules are quite complex).

ezbik commented:
https://blog.tanatos.org/posts/osfooler-ng/ installation steps on centos 7,8 and Ubuntu 18.04.

moonbaseDelta commented:
>
> @moonbaseDelta Your fork is awesome! it seems @segofensiva abandoned his development, so can you open Issues on your repo? I have some ideas:
> 

Hi, after all I could not fit that project into any usefullness and also I'm not quite sure that it worked as supposed with my previous setup. 

But I like the idea of make it done and helpful for others. 
I'm opening the comments and will try to rewrite current code on python3 soon.

moonbaseDelta committed:
Update README.md

moonbaseDelta committed:
Logo moved inside repository

moonbaseDelta committed:
Update README.md

moonbaseDelta committed:
Update README.md

moonbaseDelta committed:
Fixed a typo, and better ssh-experience now

Fixed https://github.com/moonbaseDelta/OSfooler-ng/issues/6 and added another iptables removal (should be better while stopping through ssh-session).

moonbaseDelta committed:
missindentation fixed

mvproton committed:
Change p0f database boundary

mvproton committed:
Fix p0f database parsing for last line without following delimeter

moonbaseDelta committed:
Merge pull request #15 from mvproton/master

Fix p0f database parsing

moonbaseDelta committed:
#19 mangle-table to py2-ver
